views/assets/js/App/PackageFilter.js

```diff
@@ -71,13 +74,27 @@ class PackageFilter {
             .textContent
         : elem.textContent;
 
+      // Get the "own_packages_only" field
+      let is_own = true;
+
+      if (own_packages_only) {
+        is_own = false;
+
+        window.config.own_packages.forEach((packageName) => {
+          if (filterableContent.toLowerCase().indexOf(packageName) !== -1) {
```

Comment:
Wouldn't it make more sense to explode the package name by `/`, take the first part (the vendor part) and check if that value matches exactly?

Reply:
It does, it's added now.